lmod_proxy/edx_grades/actions.py

```diff
@@ -119,3 +129,13 @@ def get_sections(gradebook, form):
         data,
         not bool(error_message)
     )
+
+
+def _get_assignment_name_max_points(form):
+    try:
+        csv_data = form.datafile.data.stream.read()
+        short_name = csv_data[0][1]
+        max_points = csv_data[1][2]
```

Comment:
@pwilkins informed me that there is always one assignment here, but we should still change this code to use `csv.DictReader` or otherwise keep track of the column header so that we don't have to hardcode the column indexes here.